examples/stream/src/main.rs

```diff
@@ -0,0 +1,140 @@
+use std::{io, time::Duration};
+
+use anyhow::{Context, Result};
+use futures::{AsyncReadExt, AsyncWriteExt, StreamExt};
+use libp2p::{multiaddr::Protocol, Multiaddr, PeerId, Stream, StreamProtocol};
+use stream_example as stream;
+use tracing::level_filters::LevelFilter;
+use tracing_subscriber::EnvFilter;
+
+const PROTOCOL: StreamProtocol = StreamProtocol::new("/my-ping-protocol");
```

Comment:
To prevent confusion with the existing `libp2p-ping`, how about implementing an _echo_ protocol in this example? On the wire, this is the same thing. But using a different name prevents users from thinking that this is **the* libp2p way to ping.

Reply:
Should the protocol then be to echo a length-prefixed message?

Reply:
Fine by me. Or the server reads until stream closing and the echoes the message.